internal/uidriver/glfw/ui.go

```diff
@@ -402,6 +402,18 @@ func (u *UserInterface) SetFullscreen(fullscreen bool) {
 	u.setWindowSize(w, h, fullscreen, u.vsync)
 }
 
+func (u *UserInterface) IsForeground() (foreground bool) {
```

Comment:
I prefer not using a named returning variable here. When the game does not start, let's return false.

```go
func (u *UserInterface) IsForeground() bool {
    if !u.isRunning() {
        return false
    }
    var foreground bool
    _ = u.t.Call(func() error {
        foreground = u.window.GetAttrib(glfw.Focused) == glfw.True
        return nil
    })
    return foreground
}
```